.storybook/FormField.js

```diff
@@ -164,6 +164,31 @@ storiesOf('FormField', module)
             </InputField>
           </Box>
         </Flex>
+        <Flex width={900}>
+          <Box width={1 / 2} pl={3}>
+            <Text bold pt={4} pb={2}>
+              Label component has hidden prop
+            </Text>
+            <InputField
+              onChange={action('onChange event is triggered in FormField')}
```

Comment:
The `onChange` event seems to be triggered in the `InputField` field, based on looking at the prop, right?